pkg/products/amqonline/reconciler.go

```diff
@@ -337,6 +343,37 @@ func (r *Reconciler) reconcileStandardAuthenticationService(ctx context.Context,
 		return integreatlyv1alpha1.PhaseFailed, fmt.Errorf("failed reconciling enmasse standard auth service keycloak secret: %w", err)
 	}
 
+	// get the secret created by the cloud resources operator
+	croSec := &corev1.Secret{}
+	err = serverClient.Get(ctx, k8sclient.ObjectKey{Name: postgres.Status.SecretRef.Name, Namespace: postgres.Status.SecretRef.Namespace}, croSec)
+	if err != nil {
+		return integreatlyv1alpha1.PhaseFailed, fmt.Errorf("failed to get postgres credential secret: %w", err)
+	}
```

Comment:
I think this fetch can replace the initial `croSecret` fetch in the code block above it
https://github.com/integr8ly/integreatly-operator/blob/5ab142cc591cdcbf4540e4b46bc26abfdf9ae000/pkg/products/amqonline/reconciler.go#L301-L307

Reply:
Is the backup secret the same as the croSecret? (I've no idea lol)

Reply:
Yup, the backup secret is the same as the `croSec`, it's just that the backup secret is in the product namespace under a different name for the backup container to use

On the above code suggestion, I just think these 2 fetches are the same, so would be good to remove of them ( `croSecret` for `croSec` fetch ) :thinking: